server/chat-plugins/datasearch.ts

```diff
@@ -146,6 +153,7 @@ export const commands: ChatCommands = {
 			`<code>Alola</code>, <code>Galar</code>, <code>Therian</code>, <code>Totem</code>, or <code>Primal</code> can be used as parameters to search for those formes.<br/>` +
 			`Parameters separated with <code>|</code> will be searched as alternatives for each other; e.g., <code>trick | switcheroo</code> searches for all Pok\u00e9mon that learn either Trick or Switcheroo.<br/>` +
 			`You can search for info in a specific generation by appending the generation to ds or by using the <code>maxgen</code> keyword; e.g. <code>/ds1 normal</code> or <code>/ds normal, maxgen1</code> searches for all Pok\u00e9mon that were Normal type in Generation I.<br/>` +
+			`You can search for info in a specific mod by using <code>mod=[mod name]</code>; e.g. <code>/nds mod=ssb, protean</code>. All valid mod names are: <code>${Object.keys(Dex.dexes).filter(x => x !== 'sourceMaps').join('</code>, <code>')}</code><br />` +
```

Comment:
This should probably be hardcoded at the top of the file so that it only iterates over `Dex.dexes` on hotpatch instead of every time someone does `/help dexsearch`.